app/controllers/users_controller.rb

```diff
@@ -17,6 +17,35 @@ def show
     @user = User.find(params[:id])
   end
 
+  def edit
+    @user = User.find(params[:id])
+    authorize! :edit, @user
+  end
+
+  def update
+    @user = User.find(params[:id])
+    authorize! :update, @user
+
+    unless user.admin?
```

Comment:
@user?

Reply:
What do you mean with that?

Reply:
I was wondering whether line 29 should read `unless @user.admin?`

Reply:
Ah ok, I see.
It is supposed to be current_user I think